src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -199,46 +199,19 @@ public VpnService.Builder newBuilder() {
    * Helper method to build a TunnelConfig from a JSON object.
    *
    * @param tunnelId unique identifier for the tunnel.
-   * @param config JSON object containing TunnelConfig values.
+   * @param configString Config string containing TunnelConfig values.
    * @throws IllegalArgumentException if `tunnelId` or `config` are null.
    * @throws JSONException if parsing `config` fails.
    * @return populated TunnelConfig
    */
-  public static TunnelConfig makeTunnelConfig(final String tunnelId, final JSONObject config)
+  public static TunnelConfig makeTunnelConfig(final String tunnelId, final String configString)
```

Comment:
Let's qualify all mentions to "config", since it's ambiguous.
```suggestion
  public static TunnelConfig makeTunnelConfig(final String tunnelId, final String proxyConfigString)
```